app/src/main/java/org/simple/clinic/home/HomeScreen.kt

```diff
@@ -99,10 +100,21 @@ class HomeScreen(context: Context, attrs: AttributeSet) : RelativeLayout(context
   }
 
   fun showOverdueAppointmentCount(count: Int) {
-    // TODO (SM): Update overdue tab badge to display the overdue count
+    val overdueTabIndex = HomeTab.values().indexOf(OVERDUE)
+    val overdueTab = homeTabLayout.getTabAt(overdueTabIndex)
+
+    overdueTab?.run {
+      val badge = orCreateBadge
```

Comment:
Ugh, it took me a while to realise that this `orCreateBadge` is actually a method, `getOrCreateBadge` which Kotlin is treating as a getter method when it is not.

Can we use the complete method call instead?

Reply:
I tried this, and added a `.apply {}` block to the badge.

```kotlin
overdueTab?.run {
  @Suppress("UsePropertyAccessSyntax")
  getOrCreateBadge().apply {
      isVisible = true
      maxCharacterCount = 3
      number = count 
  }
}
```

Reply:
Done